src/SiddharthS/Assignment2P5.java

```diff
@@ -0,0 +1,31 @@
+public class Assignment2P5
+{
+	int[] num = {2,6,3,9,11,1,20};
+	int largestNumber = num[0];
+	int secondLargestNumber = num[0];
+	
+	public static void main(String[] args)
+	{
+		new Assignment2P4().findLargestAndShortestNumInArray();
+	}
+	void findLargestAndShortestNumInArray()
+	{
+		for(int i=0;i<num.length;i++)
+		{
+			if(num[i]>largestNumber)
+			{
+				largestNumber = num[i];
+			}
+		}
+		System.out.println("Largest number in the array set is "+ largestNumber);
+		for(int i=0;i<num.length;i++)
```

Comment:
Hi @SiddharthSubham 
Here Logic is correct... but time complexity is more...
better to write this logic in single for loop....